docker-compose-non-dev.yml

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-x-superset-image: &superset-image apachesuperset.docker.scarf.sh/apache/superset:${TAG:-latest}
+x-superset-image: &superset-image apachesuperset.docker.scarf.sh/apache/superset:${TAG:-master}
```

Comment:
I think it makes sense to have devs work from master, but for production, wouldn't it be better for them to work from an official release tag?

Reply:
Agreed with @eschutho - I don't understand all of #26997 but it seems like these people are all trying to run latest builds off master instead of an official release?  Or using a mismatch of master repo clone + an official release.  Which underscores for me the value of directing new users to an official release instead of master, so that they have a positive experience right out of the box.

Reply:
See also my comment here: https://github.com/apache/superset/issues/26997#issuecomment-1956816278